assets/script.js

```diff
@@ -0,0 +1,382 @@
+// console.log("hello")
+
+// Targeting the banner box
+const bannerBox = document.getElementById("banner-box");
+
+// Targeting the start quiz button
+const startButton = document.getElementById("start-btn")
+
+// Targeting the main section
+const mainSection = document.getElementById("main")
+
+const resultDiv = document.querySelector("#result-div");
+const resultText = document.querySelector("#result-text");
+
+// const targeting the time
+const timerSpan = document.getElementById("time-span")
+
+// const formSection = document.getElementById("form-box");
+
+// question index
+let questionIndex = 0;
+let time = 60;
+let feedbackResults ={};
+let correctAnswers = 0;
+let currentQuestion
+let gameOver = false;
+
+
+const handleTimerButton = () => {
+    console.log("start button clicked");
+    const updateTimerValue = () => {
+      // increase the  timer by 1
+      time -= 1;
+  
+      // set text to new timer figures
+      timerSpan.textContent = timer;
```

Comment:
Pls, fix this... you defined time but you are calling timer and that is why the timer is not reading when the quiz starts